controllers/windowsmachine_controller.go

```diff
@@ -247,6 +247,12 @@ func (r *WindowsMachineReconciler) Reconcile(ctx context.Context, request ctrl.R
 		err := r.client.Get(ctx, kubeTypes.NamespacedName{Namespace: machine.Status.NodeRef.Namespace,
 			Name: machine.Status.NodeRef.Name}, node)
 		if err != nil {
+			// Delete a configured/running machine if no associated node can be determined (node object deleted)
```

Comment:
Is `configured` a valid  phase of machine actuation?

Reply:
It is not, I'll remove `configured` for clarity.